api-reference/10 UI Components/dxOverlay/1 Configuration/copyRootClassesToWrapper.md

```diff
@@ -5,7 +5,27 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Copies user-defined classes from the root element to the wrapper element. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The following markup illustrates relations between wrapper and root elements when you set **copyRootClassesToWrapper** to **true**:
+
+    <!--HTML--><body>
+        
+        <!-- The following is the root element on which the component is initialized. -->
+        <!-- Specify classes to be copied to the wrapper element here. -->
+        <div id="{widgetname}" class="custom-class" ... ></div>
```

Comment:
AFAIK, we don't have a `{widgetname}` placeholder.

Reply:
We've already used it here: https://github.com/DevExpress/devextreme-documentation/blob/21_1/api-reference/10%20UI%20Components/GridBase/1%20Configuration/highlightChanges.md

![image](https://user-images.githubusercontent.com/33040934/127384664-fbe008de-beb5-442f-bea3-9adaaeaeefe3.png)